agent/Dockerfile

```diff
@@ -12,12 +11,11 @@ ENV NVIDIA_VISIBLE_DEVICES=all NVIDIA_DRIVER_CAPABILITIES=utility
 # discussion and the origin of this workaround). Even when not necessary, it should be harmless.
 RUN ldconfig
 
-COPY dist/determined-agent_${VERSION}_linux_amd64.deb /tmp/agent.deb
-RUN apt-get install -y /tmp/agent.deb
-
-COPY share/determined/agent/scripts/entrypoint.sh /run/determined/workdir/entrypoint.sh
+COPY packaging/entrypoint.sh /run/determined/workdir/entrypoint.sh
 RUN chmod +x /run/determined/workdir/entrypoint.sh
 
+COPY determined-agent /usr/bin/determined-agent
```

Comment:
blocking: Why not continue to install using the deb package? It makes it clear what the environment differences are from a regular install and futureproofs against adding more files for the agent later on. And I don't see any real cost to it, since we're still building the deb for release anyway.

Reply:
We are using goreleaser to package the docker images now. The approach that they take for doing this is to link to the binary directly and to specify extra_files.

Reply:
Oh right, that was why I didn't use GoReleaser for images in the first place. I guess I'd personally still rather deal with the Docker build than have to repeat all the configuration, but I can see that going either way. :shrug: